doc/htmldoc/conf.py

```diff
@@ -247,6 +248,81 @@ def config_inited_handler(app, config):
     )
 
 
+def add_button_to_examples(app, env, docnames):
+    """Find all examples and include a link to launch notebook.
+
+     Function finds all restructured text files in auto_examples
+     and injects the multistring prolog, which is rendered
+     as a button link in HTML. The target is set to a Jupyter notebook of
+     the same name and a service to run it.
+     The nameholder in the string is replaced with the file name.
+
+     The rst files are generated at build time by Sphinx_gallery.
+     The notebooks that the target points to are linked with
+     services (like EBRAINS JupyterHub) that runs notebooks using nbgitpuller.
+     See https://hub.jupyter.org/nbgitpuller/link.html
+     The notebooks are located in the repository nest/nest-simulator-examples/.
+     The notebooks are generated from the CI workflow of NEST
+     on GitHub, which converts the source Python files to .ipynb.
+
+     The link to run the notebook is rendered in an image within a card directive.
+    """
+    example_prolog = """
+.. only:: html
+
+  .. card:: Run this example as a Jupyter notebook
+    :margin: auto
+    :width: 50%
+    :text-align: center
+
+    .. image:: https://nest-simulator.org/TryItOnEBRAINS.png
+         :target: https://lab.ebrains.eu/hub/user-redirect/git-pull?repo=\
+https%3A%2F%2Fgithub.com%2Fnest%2Fnest-simulator-examples\
+&urlpath=lab%2Ftree%2Fnest-simulator-examples%2Fnotebooks%2F\
+notebooks%2Ffilepath.ipynb&branch=main
+
+    For details and troubleshooting see :ref:`run_jupyter`."""
+
+    # Find all relevant files
+    # Inject prolog into Python example
+    files = list(Path(doc_build_dir / "auto_examples/").rglob("**/*.rst"))
```

Comment:
```suggestion
    files = list(Path(doc_build_dir / "auto_examples/").rglob(*.rst"))
```
Don't think we need the ** in rglob, only in glob?

Reply:
Yes you're right I removed the ** and reran the examples and they seem to work as expected.